src/main/java/at/ac/tuwien/kr/alpha/solver/ThriceTruth.java

```diff
@@ -57,6 +57,13 @@ public static ThriceTruth valueOf(boolean value) {
 	}
 
 	/**
+	 * @return true if this is MBT.
+	 */
+	public boolean isMBT() {
```

Comment:
My intention was to have a shorthand for MBT.equals(truth), but I think, we could replace this and all ThriceTruth.MBT/TRUE/FALSE.equals with simple == because ThriceTruth is an enum. Correct?